central/postgres/pruning.go

```diff
@@ -36,24 +37,52 @@ const (
 		(SELECT 1 FROM deployments WHERE pi.deploymentid = deployments.Id) AND 
 		(signal_time < now() at time zone 'utc' - INTERVAL '%d MINUTES' OR signal_time is NULL))`
 
+	//deleteOrphanedPLOPDeploymentsAndPI = `DELETE FROM listening_endpoints WHERE processindicatorid in (SELECT id from process_indicators pi WHERE NOT EXISTS
+	//	(SELECT 1 FROM deployments WHERE pi.deploymentid = deployments.Id) AND 
+	//	(signal_time < now() at time zone 'utc' - INTERVAL '%d MINUTES' OR signal_time is NULL) FOR UPDATE)`
+
```

Comment:
I am leaving in commented out code for now to show what alternative queries I have tried. This query probably doesn't make sense, because it looks the process_indicators table, not the listening_endpoints table. After making this mistake I fixed it for some of the queries, but not all of them. This is one query that I didn't try to fix.